docs/source/model_design/general_reference_material/ensemble_distributions/index.rst

```diff
@@ -229,6 +238,38 @@ modeling <GBD_ensemble_repo_description>` and for :ref:`Vivarium
   - `formatting.py`_ contains helper functions for formatting data and
     converting between data types.
 
+  - Note that as of July 2023 by default, :code:`risk_distributions.py` sets XMIN and XMAX 
+    parameter values to the 0.001 and 0.999 percentile values of a lognormal distribution
+    with the specified mean and standard deviation, `as shown here 
+    <https://github.com/ihmeuw/risk_distributions/blob/242c3c16c403bdf3833a7d3077bd24708150795b/src/risk_distributions/risk_distributions.py#L76>`_
+
+.. note::
+
+  Given that there are two different code bases meant to perform the same task, we have
+  verified that the two code bases indeed accurately replicate one another `in a notebook
+  that can be found here <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/blob/data_prep/data_prep/risk_distribution/risk%20distribution%20comparison.ipynb>`_. 
+  We found that the two code bases do accurately replicate one another with a few exceptions:
```

Comment:
Same comment that I think we want our code base to replicate the GBD one rather than them replicating one another.